CHANGELOG.md

```diff
@@ -6,6 +6,462 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Wellness ToDo List - UI updates (Task in progress) [#1689](https://github.com/rokwire/illinois-app/issues/1689).
+
+## [4.0.30] - 2022-06-15
+### Changed
+- Gies content updated  [#1749](https://github.com/rokwire/illinois-app/issues/1749).
+- Cleaned up FlexUI rules [#1742](https://github.com/rokwire/illinois-app/issues/1742).
+- Updated icnos from Zeplin desgin [#1748](https://github.com/rokwire/illinois-app/issues/1748).
+### Added
+- Wellness Resources drop down item [#1741](https://github.com/rokwire/illinois-app/issues/1741).
+- Wellness tip background color is color of the day [#1746](https://github.com/rokwire/illinois-app/issues/1746).
+
+## [4.0.29] - 2022-06-14
+### Changed
+- Default drop down selection in Academics panel [#1726](https://github.com/rokwire/illinois-app/issues/1726).
+- Favorites panel UI changes [#1729](https://github.com/rokwire/illinois-app/issues/1729).
+- Updated tab icons [#1737](https://github.com/rokwire/illinois-app/issues/1737).
+- Texts in Favorites panel [#1736](https://github.com/rokwire/illinois-app/issues/1736).
+### Added
+- Added "See All" to all home list widgets, openes the corresponding content panel [#1727](https://github.com/rokwire/illinois-app/issues/1727).
+- Added "WPGU FM Radio" button to header bar title if radio is playing [#1731](https://github.com/rokwire/illinois-app/issues/1731).
+- Added more entries to different browse sections [#1733](https://github.com/rokwire/illinois-app/issues/1733).
+
+## [4.0.28] - 2022-06-13
+### Added
+- Created Recent Items panel, linked to Browse panel [#1701](https://github.com/rokwire/illinois-app/issues/1701).
+- Created HomeWelcomeWidget [#1718](https://github.com/rokwire/illinois-app/issues/1718).
+- Added Favorite buttons to HomeWalletWidget items [#1718](https://github.com/rokwire/illinois-app/issues/1718).
+### Changed
+- Cleaned up recent items handling [#1708](https://github.com/rokwire/illinois-app/issues/1708).
+- Better processing of Home editing headers as drop targets [#1718](https://github.com/rokwire/illinois-app/issues/1718).
+- Updated Wellness health rings (Work in progress)  [#1692](https://github.com/rokwire/illinois-app/issues/1692).
+- Academics Panel UI [#1714](https://github.com/rokwire/illinois-app/issues/1714).
+### Fixed
+- Fixed HomeFavoritesWidget titles for Notifications and Guide Items [#1718](https://github.com/rokwire/illinois-app/issues/1718).
+- Store user's drop down selection in each panel [#1721](https://github.com/rokwire/illinois-app/issues/1721).
+
+## [4.0.27] - 2022-06-10
+### Added
+- Content for Academics Panel [#1701](https://github.com/rokwire/illinois-app/issues/1701).
+- Store user selection for Settings, Profile, Notifications and Wellness [#1706](https://github.com/rokwire/illinois-app/issues/1706).
+### Changed
+- Handled properly different "My ___" types from Browse panel [#1705](https://github.com/rokwire/illinois-app/issues/1705).
```

Comment:
It looks like an escape character is missing something related. Adding this comment for future reference and fixing. Thanks.